pages/restake/lrt-concepts.mdx

```diff
@@ -0,0 +1,53 @@
+# Liquid Restaking on Tangle Network
+
+## Introduction
+
+Liquid restaking allows users to participate in staking while still maintaining liquidity of their staked assets.
```

Comment:
This is wrong. It allows participating in restaking while maintaining liquidity.